addons/website/views/website_templates.xml

```diff
@@ -990,36 +966,38 @@
                 <!-- Navbar Collapse -->
                 <div id="top_menu_collapse" class="collapse navbar-collapse">
                     <t t-call="website.navbar_nav">
-                        <t t-set="_nav_class" t-value="'ml-3 flex-grow-1'"/>
+                        <t t-set="_nav_class" t-valuef="ml-3 flex-grow-1"/>
                         <!-- Menu -->
                         <t t-foreach="website.menu_id.child_id" t-as="submenu">
                             <t t-call="website.submenu">
-                                <t t-set="item_class" t-value="'nav-item'"/>
-                                <t t-set="link_class" t-value="'nav-link'"/>
+                                <t t-set="item_class" t-valuef="nav-item"/>
+                                <t t-set="link_class" t-valuef="nav-link"/>
                             </t>
                         </t>
                         <!-- Sign In -->
                         <t t-call="portal.placeholder_user_sign_in">
-                            <t t-set="_item_class" t-value="'nav-item ml-lg-auto'"/>
-                            <t t-set="_link_class" t-value="'nav-link'"/>
+                            <t t-set="_item_class" t-valuef="nav-item ml-lg-auto"/>
+                            <t t-set="_link_class" t-valuef="nav-link"/>
                         </t>
                         <!-- User Dropdown -->
                         <t t-call="portal.user_dropdown">
                             <t t-set="_icon" t-value="True"/>
                             <t t-set="_user_name" t-value="True"/>
-                            <t t-set="_item_class" t-value="'nav-item dropdown ml-lg-auto'"/>
-                            <t t-set="_link_class" t-value="'nav-link'"/>
+                            <t t-set="_item_class" t-valuef="nav-item dropdown ml-lg-auto"/>
+                            <t t-set="_link_class" t-valuef="nav-link"/>
+                        </t>
+                        <!-- Language Selector -->
+                        <t t-call="website.placeholder_header_language_selector">
+                            <t t-set="_div_classes" t-valuef="ml-lg-2 ml-1 align-self-lg-center"/>
                         </t>
-                        <!-- Button -->
                         <li class="nav-item">
-                            <div class="oe_structure oe_structure_solo" id="oe_structure_header_image_2"/>
+                            <!-- Call To Action -->
+                            <t t-call="website.placeholder_header_call_to_action">
+                                <t t-set="_div_classes" t-valuef="d-none d-lg-block"/>
+                            </t>
```

Comment:
Should not the d-none class be used on the parent nav-item instead of the div inside the t-call?